src/components/Article.jsx

```diff
@@ -0,0 +1,42 @@
+import ArticleInfo from "./ArticleInfo"
+import { useParams } from "react-router-dom"
+import axios from "axios"
+import { useEffect, useState} from "react"
+
+
+export default function Article() {
+    const {article_id} = useParams()
+
+
+    let [article, setArticle] = useState()
+    let [loadingArticle, setLoadingArticle] = useState(true)
+    let [comments, setComments] = useState()
+
+    useEffect(() => {
+        setLoadingArticle(true)
+        axios.get(`https://nc-news-backend-shahbazrafi.herokuapp.com/api/articles/${article_id}`)
+        .then(({data}) => {
+            setArticle(data.articles)
+            setLoadingArticle(false)
+        })
+        .catch(err => console.log("error", err))
+    }, [article_id]) 
+
+    useEffect(() => {
+        axios.get(`https://nc-news-backend-shahbazrafi.herokuapp.com/api/articles/${article_id}/comments`)
```

Comment:
Same as above, don't need comment api call yet so shouldn't be included in this PR